### ISSUE TITLE
Brownstone Roads broken - update 2.0.1.3

### ISSUE THREAD

**Issue by tranceplanet** [GTNewHorizons/NewHorizons#2128]
#### Which modpack version are you using?
GT New Horizons 2.0.1.3
#
#### If in multiplayer; On which server does this happen? 
Dirt Huts
#
#### What did you try to do, and what did you expect to happen?
Prior to 2.0.1.3 I'm using Brownstone roads as my main source of pathways and they are littered across my town.
#
#### What happend instead? (Attach screenshots if needed)
Since the update; Brownstone now acts like random directional ice and it is impossible to walk in a straight line or stand still.  (High speed inertia drift?)
#
#### What do you suggest instead/what changes do you propose?
Fix

**Comment by Dream-Master** [comment]
@Technus 
Are there some code changes that cause this issue ?

**Comment by tranceplanet** [comment]
Also note:
I have noticed 'sliding' on top of Tinkers Crafting Tables and Seared Stone also like ice.  This is only what I have noticed so far so there could be more.  This could be something affecting Tinkers Construct blocks only as Brownstone is also Tinkers.

**Comment by Dream-Master** [comment]
Yes i heard about that this is a undocumented feature but brownstone was not changed or any other code in Tinkers.

**Comment by Technus** [comment]
Interesting.

**Comment by tranceplanet** [comment]
Just an update:
All Tinker's Construct blocks act like ice from what I've noticed so far.

Apparently before 2.0.1.3 it **was** existing, but only a few people had it.

After 2.0.1.3, it appears to affect everybody oddly.

Seems a bit of a pain for some who have designed their paths with Tinker's but for me I have already converted to GregTech Concrete and don't worry about it anymore.

Of low concern for me personally.
